pkg/controller/operators/olm/requirements.go

```diff
@@ -14,6 +14,8 @@ import (
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
```

Comment:
nit: unconventional import order -- looks like it was already a problem before this patch. Would appreciate it if you could fix here.

We use goimports for formatting, but in general we split imports into three groups, like so:

```go
import (
        "encoding/json"
  	"fmt"
	"strings"

	"github.com/sirupsen/logrus"
	"github.com/coreos/go-semver/semver"
	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"

	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
	"github.com/operator-framework/api/pkg/operators/v1alpha1"
	olmErrors "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/errors"
	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/install"
)
```

Reply:
Thanks, @njhale nice catch, I use `go fmt` to format the import order.